definitions.go

```diff
@@ -93,21 +95,39 @@ type MethodArgument struct {
 
 type RefType struct {
 	Name string
-	Pkg  []*Package
-	Type []Type
+	Pkg  *Package
+	Type Type
 }
 
 func NewRefType(pkg *Package) *RefType {
-	ref := &RefType{}
-	ref.Pkg[0] = pkg
-	return ref
+	return &RefType{
+		Pkg: pkg,
+	}
 }
 
 type Tag struct {
 	Raw    string
 	Params []TagParam
 }
 
+func (t *Tag) AppendTagParam(tNew *TagParam) bool {
+	_, ok := t.TagParamByName(tNew.Name)
+	if ok {
+		return !ok
+	}
+	t.Params = append(t.Params, *tNew)
+	return ok
```

Comment:
Isn't this going to be always false at this point? Is that the desired behavior?

Reply:
False inform you that there was no problem in add new value, however, the return type should change to error. Thx 😁

Reply:
This has been resolved in branch **builtin-parse**. I believe than not is necessary return error, so, i changed func to resolve problem with return. (always returning false).